### PR TITLE
win: patch for 531.61

### PR DESCRIPTION
**Pull request by Umgak**
This pull adds support for NVIDIA GeForce driver 531.61.
Generated via autopatch.py, verified and appears to work via:

```
ffmpeg.exe -y -hwaccel cuda -hwaccel_output_format cuda `
-f lavfi -i testsrc -t 50 `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 4M -f null - `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 1M -f null - `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 8M -f null - `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 6M -f null - `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 5M -f null - `
-vf hwupload -fps_mode passthrough -c:a copy -c:v h264_nvenc -b:v 7M -f null -  
ffmpeg.exe : ffmpeg version 2022-09-22-git-af919cf780-full_build-www.gyan.dev Copyright (c) 2000-2022 the FFmpeg 
developers
  built with gcc 12.1.0 (Rev2, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect 
--enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma 
--enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth 
--enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libdav1d --enable-libdavs2 
--enable-libuavs3d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 
--enable-libx265 --enable-libxavs2 --enable-libxvid --enable-libaom --enable-libjxl --enable-libopenjpeg 
--enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi 
--enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm 
--enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx 
--enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme 
--enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine 
--enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm 
--enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b 
--enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      57. 36.102 / 57. 36.102
  libavcodec     59. 44.100 / 59. 44.100
  libavformat    59. 32.100 / 59. 32.100
  libavdevice    59.  8.101 / 59.  8.101
  libavfilter     8. 49.100 /  8. 49.100
  libswscale      6.  8.112 /  6.  8.112
  libswresample   4.  9.100 /  4.  9.100
  libpostproc    56.  7.100 / 56.  7.100
Input #0, lavfi, from 'testsrc':
  Duration: N/A, start: 0.000000, bitrate: N/A
  Stream #0:0: Video: wrapped_avframe, rgb24, 320x240 [SAR 1:1 DAR 4:3], 25 fps, 25 tbr, 25 tbn
Stream mapping:
  Stream #0:0 -> #0:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
  Stream #0:0 -> #1:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
  Stream #0:0 -> #2:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
  Stream #0:0 -> #3:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
  Stream #0:0 -> #4:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
  Stream #0:0 -> #5:0 (wrapped_avframe (native) -> h264 (h264_nvenc))
Press [q] to stop, [?] for help
Output #0, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #0:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 4000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/4000000 buffer size: 8000000 vbv_delay: N/A
Output #1, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #1:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 1000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/1000000 buffer size: 2000000 vbv_delay: N/A
Output #2, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #2:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 8000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/8000000 buffer size: 16000000 vbv_delay: N/A
Output #3, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #3:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 6000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/6000000 buffer size: 12000000 vbv_delay: N/A
Output #4, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #4:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 5000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/5000000 buffer size: 10000000 vbv_delay: N/A
Output #5, null, to 'pipe:':
  Metadata:
    encoder         : Lavf59.32.100
  Stream #5:0: Video: h264 (Main), cuda(pc, gbr/unknown/unknown, progressive), 320x240 [SAR 1:1 DAR 4:3], q=2-31, 7000 
kb/s, 25 fps, 25 tbn
    Metadata:
      encoder         : Lavc59.44.100 h264_nvenc
    Side data:
      cpb: bitrate max/min/avg: 0/0/7000000 buffer size: 14000000 vbv_delay: N/A
frame=    0 fps=0.0 q=0.0 q=0.0 q=0.0 q=0.0 q=0.0 q=0.0 size=       0kB time=-577014:32:22.77 bitrate=  -0.0kbits/s 
speed=N/A    
frame=  215 fps=0.0 q=9.0 q=9.0 q=9.0 q=9.0 q=9.0 q=9.0 size=N/A time=00:00:08.44 bitrate=N/A speed=16.9x    
```

### CODE REVIEW

File: README.md
Comment (jailuthra) [inline README.md:1]: Looks like windows README has overwritten the root (linux) README.

Can you revert the changes to this file and use the readme-autogen tool to update the README?

**Review by jailuthra — CHANGES_REQUESTED** [review]
Thanks for the PR, patch looks good. README needs some fixes.

**Comment by Umgak** [comment]
Should be fixed now, sorry about that!

**Comment by jailuthra** [comment]
No worries, thank you for the contribution!

**Review by jailuthra — APPROVED** [review]
